internal/handlers/handlers.go

```diff
@@ -0,0 +1,133 @@
+package handlers
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/Dreeedy/shorturl/internal/config"
+	"github.com/Dreeedy/shorturl/internal/storage"
+	"github.com/go-chi/chi"
+	"golang.org/x/exp/rand"
```

Comment:
В даннгом случае достаточно использовать `math/rand`. В этом случае `Seed` вызывается автоматически, потому вручную его вызывать не надо (см. https://pkg.go.dev/math/rand#Seed)

Reply:
Поправил.

Теперь "golang.org/x/exp/rand" не используется.

Reply:
Приходится использовать "crypto/rand" т.к. "math/rand" приводил к ошибке в статическом тесте internal/handlers/handlers.go:108:15: rand.Read has been deprecated since Go 1.20 because it shouldn't be used: For almost all use cases, [crypto/rand.Read] is more appropriate.